Soccar [UI]/src/nl/socnet/message/handler/BallSyncMessageHandler.java

```diff
@@ -0,0 +1,46 @@
+package nl.socnet.message.handler;
+
+import io.netty.buffer.ByteBuf;
+import nl.soccar.physics.GameEngine;
+import nl.soccar.socnet.connection.Connection;
+import nl.soccar.socnet.message.MessageHandler;
+import nl.soccar.ui.Main;
+import nl.soccar.ui.fx.controller.GameViewFXMLController;
+import nl.socnet.message.BallSyncMessage;
+
+public final class BallSyncMessageHandler extends MessageHandler<BallSyncMessage> {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)